install.sh

```diff
@@ -215,6 +216,7 @@ if [ ! -f $BUILD/progress/ghcjs ]; then
   run $BUILD       rm -rf ghcjs
   run $BUILD       git clone --branch ghc-8.6 --single-branch https://github.com/ghcjs/ghcjs.git
   run $BUILD/ghcjs git submodule update --init --recursive
+  run $BUILD/ghcjs git checkout e87195eaa2bc7e320e18cf10386802bc90b7c874
```

Comment:
Is this the right order? I would have thought we want to checkout the right commit first, then run `git submodule update`, so that if ghcjs later changes the submodule version, we wouldn't build an old ghcjs against a newer submodule?

But I don't really understand git.  If you know what you're doing, ignore me.

Reply:
You are right, what I had didn't checkout the submodules. Since the git version may not be recent enough for the nice single command invocation it is a little long-winded.